frontend/src/features/public-form/PublicFormService.ts

```diff
@@ -25,13 +25,15 @@ import {
 import { FormFieldValues } from '~templates/Field'
 
 import {
-  createEmailSubmissionFormData,
+  createClearSubmissionFormData,
   createEncryptedSubmissionData,
 } from './utils/createSubmission'
 import { filterHiddenInputs } from './utils/filterHiddenInputs'
 
 export const PUBLIC_FORMS_ENDPOINT = '/forms'
 
+const ENCRYPTION_BOUNDARY_SHIFT_ENCRYPTION_VERSION = 2
```

Comment:
For context - suggest just add a comment here about what the encryption boundary shift was about + link to PR number, so in future we keep context of what encryption boundary shift was all about

Reply:
comment added in [f77be1e](https://github.com/opengovsg/FormSG/pull/6665/commits/f77be1e870a75475f177caf1bfe17f7ff26453d8)